docs/about.md

```diff
@@ -5,21 +5,56 @@ hide:
 
 # Data Science in NHS England
 
-We are the NHS England Data Science Team with a vision: 
+<div markdown>
+
+![Image title](../images/DS_team_photo_smaller.jpeg){ width="450" alt-tex="Picture of the Data Science team stood on some steps in London."  align=right }
+
+
+We are the [NHS England](https://www.england.nhs.uk/) Data Science Team. 
+
+**Our vision is**: 
 
 > For NHS England to lead on embracing data science in the NHS for the betterment of all our patients and all our staff​. 
 
-<img src=../images/DS_team_photo.jpeg alt-tex="Picture of the Data Science team stood on some steps in London." width="600" >
-</img>
+<br/>
+
+[Contact Us](mailto:datascience@nhs.net){ .md-button .md-button--primary }
```

Comment:
this button doesnt do anything when you click it :'(

Reply:
Resolved to some extent!